src/Yoti/Util/Age/AbstractAgeProcessor.php

```diff
@@ -2,40 +2,85 @@
 
 namespace Yoti\Util\Age;
 
+use Yoti\Entity\Attribute;
+use Yoti\Entity\AgeVerification;
+
 abstract class AbstractAgeProcessor
```

Comment:
We now have both an _AbstractAgeProcessor_ and an _AgeProcessor_.
Might be a little clearer if this one were to become _AgeVerificationFactory_ (or _BaseAgeVerificationFactory_, or _AgeVerificationConverter_ etc) and the two sublasses would be _AgeOverVerificationFactory_ + _AgeUnderVerificationFactory_.

Reply:
I see your point, I will call it _AgeVerificationConverter_, but I will rather call the two subclasses _AgeOverVerificationProcessor_ and _AgeUnderVerificationProcessor_ as firstly it's not only creating it's also processing data, secondly the name is inline with the _AbstractAgeProcessor_.